src/InteractiveWindow/Editor/Commands/InteractiveWindowCommands.cs

```diff
@@ -271,6 +277,13 @@ public void DisplayHelp()
                 _window.Write(HelpIndent);
                 _window.WriteLine(line);
             }
+
+            _window.WriteLine(InteractiveWindowResources.ScriptDirectives);
+            foreach (var line in s_scriptDirectives)
+            {
+                _window.Write(HelpIndent);
+                _window.WriteLine(line);
+            }
```

Comment:
As per offline discussion, let's check the elevator to see it's type and if it is CSharpInteractiveEvaluator or VisualBasicInteractiveEvaluator then display the new help.